src/device-manager/java/src/nl/Weave/DataManagement/WdmClientImpl.java

```diff
@@ -135,6 +142,13 @@ public void setCompletionHandler(CompletionHandler compHandler)
         mCompHandler = compHandler;
     }
 
+    @Override
+    public String getEvents()
```

Comment:
Please add test in 
https://github.com/openweave/openweave-core/blob/master/src/device-manager/java/src/nl/Weave/DeviceManager/TestMain.java